lemmy_utils/src/request.rs

```diff
@@ -58,7 +58,7 @@ pub(crate) async fn fetch_iframely(
   client: &Client,
   url: &str,
 ) -> Result<IframelyResponse, LemmyError> {
-  let fetch_url = format!("http://iframely/oembed?url={}", url);
+  let fetch_url = format!("http://localhost:8061/oembed?url={}", url);
```

Comment:
What should this be set to? There isn't an `iframely_url` option in the Settings struct, should one be added?

Reply:
Yep this will require a new setting.